ruby.rb

```diff
@@ -0,0 +1 @@
+variable = 'This is a line that is really long and Rubocop should probably pick it up.'
```

Comment:
Useless assignment to variable - variable.<br>Missing frozen string literal comment.<br>Line is too long. [87/80]